tests/integration_tests/functional/test_metrics.py

```diff
@@ -44,6 +44,7 @@ def test_flush_metrics(test_microvm_with_api):
         'vmm',
         'uart',
         'signals',
+        'vsock'
```

Comment:
Is it necessary to have a separate commit for this change? Doesn't look like a separate functionality or something close to it.

Reply:
Squashed both commits.